Foxtrot/Driver/Rewriting/PostRewriteChecker.cs

```diff
@@ -141,10 +141,7 @@ public override void VisitMemberBinding(MemberBinding binding)
                 methodName.Matches(ContractNodes.EndContractBlockName))
             {
                 string message = string.Format(Resources.Error_ContractNotRewritten_ContractName_MemberName, method.FullName, this.visitedMembers.Peek().FullName);
-                if (this.handleError != null)
-                {
-                    this.handleError(new Error(1080, message, binding.SourceContext));
-                }
+                this.handleError(new Error(1080, message, binding.SourceContext));
```

Comment:
There's an invariant on it as non-null. It's also only set from the constructor, which has a precondition for non-null.